packages/coding-standards/composer.json

```diff
@@ -16,7 +16,7 @@
     "require": {
         "php": "^7.2",
         "ext-tokenizer": "*",
-        "friendsofphp/php-cs-fixer": "^2.14.0",
+        "friendsofphp/php-cs-fixer": "^2.14.0 !=2.19.0",
```

Comment:
wouldn't be better to configure it in `conflicts` section?

Reply:
Yeah, you are right, thanks!